src/common/Args.cpp

```diff
@@ -147,7 +147,7 @@ void Args::applyCustomChannelLayout(const QString &argValue)
 
             // Set first tab as selected
             tab.selected_ = window.tabs_.empty();
-            tab.rootNode_ = SplitNodeDescriptor{"twitch", channelName};
+            tab.rootNode_ = SplitNodeDescriptor{{"twitch", channelName}};
```

Comment:
Can you please explain this change?

Reply:
I explained it in ef292c6's commit message and the PR - Qt Creator thown a warning saying we should wrap this inside braces.

Reply:
i see